source/manuals/programming-languages/nodejs/index.html.md.erb

```diff
@@ -253,6 +253,75 @@ const callback = filename => (err, data) => {
 };
 ```
 
+#### Use of `async`/`await`
+
+If your version of Node is above 7.10.0 you should use async functions, and the await keyword, to organise your asynchronous code. This will make it easier to read and write.
```

Comment:
```suggestion
If your version of Node is above 7.10.0 you should use async functions, and the await keyword, to organise your asynchronous code. This will make your code easier to read and write.
```